### PR TITLE
Remove ddtrace.util.py

### PR DESCRIPTION
**Pull request by ZStriker19**
## Commit Message
Remove ddtrace.util.py 

Removing ddtrace.util.py for 1.0. It's already been deprecated.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: releasenotes/notes/removed_ddtrace.util.py-0d0f48aefa6d9779.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed ``ddtrace/util.py``. Please use modules in ddtrace/utils instead.
```
Comment (brettlangdon) [inline releasenotes/notes/removed_ddtrace.util.py-0d0f48aefa6d9779.yaml:4]:
```suggestion
    Removed ``ddtrace.util`` module. Please use ``ddtrace.utils`` instead.
```

? not sure.

**Review by brettlangdon — DISMISSED** [review]

**Review by mabdinur — DISMISSED** [review]

File: releasenotes/notes/removed_ddtrace.util.py-0d0f48aefa6d9779.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed ``ddtrace.util`` module. Please use ``ddtrace.utils`` instead.
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/removed_ddtrace.util.py-0d0f48aefa6d9779.yaml:4]: We're getting rid of ddtrace.utils as well. We should just say it's being removed. No one should be using anything from here anymore anyway

**Review by Kyle-Verhoog — CHANGES_REQUESTED** [review]

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — DISMISSED** [review]

File: docs/spelling_wordlist.txt [hunk 3]
```diff
@@ -149,6 +149,7 @@ unvendored
 url
 urls
 username
+utils
```
Comment (Kyle-Verhoog) [inline docs/spelling_wordlist.txt:152]: Probably don't need this anymore

Comment (brettlangdon) [reply docs/spelling_wordlist.txt:152]:
```suggestion
```

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**